tools/jenkins/run_distribution.sh

```diff
@@ -66,67 +66,61 @@ if [ "$platform" == "linux" ]; then
 elif [ "$platform" == "macos" ]; then
 
   if [ "$dist_channel" == "homebrew" ]; then
-    # system installed homebrew, don't interfere
+    # System installed homebrew, don't interfere. Run this again at the end
+    # to make sure the same formulas were still installed
     brew list -l
```

Comment:
Ah, now I see what you mean. I recall the thing you did on `run_jenkins.sh` now.